backend/src/main/kotlin/com/infowings/catalog/data/guid/GuidService.kt

```diff
@@ -0,0 +1,101 @@
+package com.infowings.catalog.data.guid
+
+import com.infowings.catalog.common.*
+import com.infowings.catalog.common.objekt.PropertyUpdateResponse
+import com.infowings.catalog.common.objekt.Reference
+import com.infowings.catalog.data.Subject
+import com.infowings.catalog.data.aspect.toAspectPropertyVertex
+import com.infowings.catalog.data.aspect.toAspectVertex
+import com.infowings.catalog.data.objekt.*
+import com.infowings.catalog.data.reference.book.toReferenceBookItemVertex
+import com.infowings.catalog.data.subject.toSubject
+import com.infowings.catalog.data.subject.toSubjectVertex
+import com.infowings.catalog.storage.*
+import com.orientechnologies.orient.core.record.ODirection
+
+data class EntityMetadata(val guid: String, val entityClass: String, val id: String)
+
+class GuidService(private val db: OrientDatabase, private val dao: GuidDaoService) {
```

Comment:
It would also be nice right now to have a method that would find object values. According to #253 we have to be able to retrieve `propertyName?`, `aspectName`, `objectValueData`, `measure?` attributes of value by GUID.